webapp/src/js/controllers/tasks-content.js

```diff
@@ -168,6 +168,10 @@ angular.module('inboxControllers').controller('TasksContentCtrl',
     $scope.formId = null;
     $scope.setSelected($state.params.id);
 
+    $scope.$on('ClearSelected', () => {
```

Comment:
This code existed before https://github.com/medic/medic/blob/1cdffcdc07579f3f7616f7e8cc08ae382b8bada4/webapp/src/js/controllers/tasks-content.js#L113 but was incorrectly refactored in https://github.com/medic/medic/commit/264f76404991b14609178989aa07c216279f93c2#diff-2a663112a2d4e7ebb886d473db90dd64R200 .